client/src/components/BuildingStatsTable.tsx

```diff
@@ -208,8 +225,11 @@ const BuildingStatsTableWithoutI18n = (props: { addr: AddressRecord }) => (
         <EvictionFilings />
         <EvictionsExecuted />
       </div>
+      <div className="table-row timeline-link">
+        <TimelineLink url={props.timelineUrl} />
```

Comment:
@austensen if you pull this branch, there are red underlines under the `url` variable about mismatched types. would love your eyes on this bc i think it's a simple thing i'm missing

Reply:
It need `props: {}` in the args of the functional component - just pushed an update.